go.mod

```diff
@@ -254,7 +254,9 @@ require (
 )
 
 replace (
-	github.com/cosmos/cosmos-sdk => github.com/celestiaorg/cosmos-sdk v1.28.2-sdk-v0.46.16
+	// github.com/cosmos/cosmos-sdk => github.com/celestiaorg/cosmos-sdk v1.28.2-sdk-v0.46.16
+	// TODO: Release tag: celestiaorg/cosmos-sdk release/v0.46.x-celestia
```

Comment:
Blocking on this

Reply:
Created https://github.com/celestiaorg/cosmos-sdk/releases/tag/v1.29.0-sdk-v0.50.12-rc1 for use here

Reply:
Should I create a celestiaorg/cosmos-sdk release from v0.46.x to unblock this PR?

Reply:
yes please!

Reply:
Created https://github.com/celestiaorg/cosmos-sdk/releases/tag/v1.30.0-sdk-v0.46.16